editor/action_map_editor.cpp

```diff
@@ -578,7 +574,7 @@ ActionMapEditor::ActionMapEditor() {
 	action_tree->connect("button_clicked", callable_mp(this, &ActionMapEditor::_tree_button_pressed));
 	main_vbox->add_child(action_tree);
 
-	action_tree->set_drag_forwarding_compat(this);
+	action_tree->set_drag_forwarding(callable_mp(this, &ActionMapEditor::get_drag_data_fw).bind(action_tree), callable_mp(this, &ActionMapEditor::can_drop_data_fw).bind(action_tree), callable_mp(this, &ActionMapEditor::drop_data_fw).bind(action_tree));
```

Comment:
If the syntax is always like this, maybe we could have a macro that sets up all three at once?
`SETUP_DRAG_FW(action_tree, ActionMapEditor);`

Reply:
Not all forwardings use all methods, so we'd need at least 2 (or 3?) macros. Also macro will make it more difficult to remove unnecessary `bind()`s.

But I can do that.

Reply:
Yeah if it's not always the same with three methods and a bind of the base object it's maybe not worth macroing.

It's just a pretty verbose syntax compared to the previous one.

Maybe stylistically putting each callable_mp on a new line would help with readability at least.

Reply:
Ok, I changed it to macro. I needed 4 of them, but it's cleaner and less error-prone.